### DISCUSSION TITLE
Specify custom folder in remote

### DISCUSSION THREAD

**Discussion by shahamran** [amitds1997/remote-nvim.nvim#122] [Q&A]
Hi, any way to customize the location in which remote-nvim bootstraps itself in the remote machine?

Currently it's in `~/.remote-nvim`, but my remote's home dir is on a slow filesystem (NFS), so I want to change that.

Thanks.

**Comment by shahamran** [comment]
Ok, I just realized I can do that by opening the workspace config json and editing `remote_neovim_home` variable.

Any chance to get a command to open the workspace config file in the local machine?

**Reply by amitds1997** [reply]
I plan to build a single UI where all this is configurable and instead of multiple-popups, you just fill a form using _probably [this](https://nui-components.grapp.dev/docs/components/form)_. But for now, you can just use this.

```lua
vim.cmd("e ".. require("remote-nvim").session_provider:get_config_provider()._config_path:absolute())
```

I will pick this up after completing the devcontainer feature. Unless I start work on the UI after that.
